client/apis/volumegroupsnapshot/v1beta2/types.go

```diff
@@ -194,6 +194,7 @@ type VolumeGroupSnapshotClass struct {
 	// "Delete" means that the VolumeGroupSnapshotContent and its physical group
 	// snapshot on underlying storage system are deleted.
 	// Required.
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="deletionPolicy is immutable once set"
```

Comment:
I think we do allow users to change the deletion policy for SC.

Reply:
hm ok I do see it is immutable for SC: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/storage/validation/validation.go#L79

But do we actually want that behavior?

Reply:
Yes, we don't want to change deletion policy of a VolumeGroupSnapshotClass.
We only want to change deletion policy of a VolumeGroupSnapshotContent.